test/csi-volumes.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	. "github.com/onsi/ginkgo"
+	_ "github.com/onsi/gomega"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/framework/testfiles"
+	"k8s.io/kubernetes/test/e2e/storage/testsuites"
+	"k8s.io/kubernetes/test/e2e/storage/utils"
+	"path"
+)
+
+var CSITestSuites = []func() testsuites.TestSuite{
+	testsuites.InitVolumesTestSuite,
+	testsuites.InitVolumeIOTestSuite,
+	testsuites.InitVolumeModeTestSuite,
+	testsuites.InitSubPathTestSuite,
+	testsuites.InitProvisioningTestSuite,
```

Comment:
Would it be useful to define a global variable in the test framework that contains all the test suites? That way drivers don't need to manually copy them?

Reply:
Yes I think that would be useful. Would there be any case where a driver would not want to run some of these test suites?

Reply:
It shouldn't need to. The test cases are supposed to check for driver capabilities and only run the tests that the driver can support.

Reply:
We also have Snapshot and MultiVolume test suites, although they don't seem applicable to this nfs driver. Should be include them anyway and comment them out?

Reply:
Ya sure, since this would be used as an example right. Also, I think MultiVolume testsuite is not in 1.14?

Reply:
I guess it doesn't matter since it'll still be commented out.